include/swift/AST/NameLookup.h

```diff
@@ -278,7 +278,11 @@ enum class DeclVisibilityKind {
   ///   }
   /// }
   /// \endcode
-  MemberOfProtocolImplementedByCurrentNominal,
+  MemberOfProtocolNotImplementedByCurrNominal,
```

Comment:
Nit: I'm not a fan of this name change here, but then again the original name wasn't great. I think it should stay as it is, because `NotImplemented` doesn't immediately read like you mean it to ("this is a member of a protocol, but that member is not implemented by the nominal type that is the subject of this lookup"). It reads as a contradiction: "this is a member of a protocol that the current nominal type doesn't implement", which is weird.

Reply:
Ah, I see now. Thanks!

Reply:
@CodaFi How do you like `MemberOfProtocolConformedToByCurrentNominal`?